lib/product.rb

```diff
@@ -0,0 +1,21 @@
+class Product
+  attr_reader :category, :name, :unit_price, :quantity
```

Comment:
A minor nitpick but ruby convention would have each of these attributes on a new line so
```
 attr_reader :category, 
                       :name, 
                       :unit_price, 
                       :quantity
```